lib/pg_ha_migrations/safe_statements.rb

```diff
@@ -221,16 +251,23 @@ def safe_add_concurrent_partitioned_index(
     # TODO: take out ShareLock after issue #39 is implemented
     safely_acquire_lock_for_table(parent_table.fully_qualified_name) do
       # CREATE INDEX ON ONLY parent_table
-      unsafe_add_index(
-        parent_table.fully_qualified_name,
-        columns,
+      options = {
         name: parent_index.name,
         if_not_exists: if_not_exists,
         using: using,
         unique: unique,
         where: where,
         comment: comment,
         algorithm: :only, # see lib/pg_ha_migrations/hacks/add_index_on_only.rb
+      }
+
+      # Only include nulls_not_distinct option when supported (we already checked at the top of the method)
+      options[:nulls_not_distinct] = nulls_not_distinct unless nulls_not_distinct.nil?
```

Comment:
With the check at the top of the method, I think we can just pass this through now without all this logic?

Reply:
Because of removing 7.0 support, yes.